### DISCUSSION TITLE
Including images on pane

### DISCUSSION THREAD

**Discussion by edgarjoserojaslopez** [segmentio/evergreen#1092] [General] [answered]
Hi there!! My question is: How can i insert an image or icon in pane component?

**Comment by cdedreuille** [comment, marked as answer]
Hi @edgarjoserojaslopez, The way I do it for now is like this:
```
<Pane is='img' src='path/to/your/image.jpg' alt='Image' />
```
Not sure if this is the right way, but it works for me.

**Reply by mshwery** [reply]
Yep! You can also use `Image` if you don't need Pane specifically: 

```jsx
import { Image } from 'evergreen-ui'

<Image src="pupdanceparty.gif" alt="Puppies dancing" />
```

Either approach (`Image` or `Pane is="img"` which is what Image does under the hood) will work fine _and_ you get all of the ui-box css props available directly on the component: 
```jsx
<Image src="pupdanceparty.gif" alt="Puppies dancing" marginTop={32} />
```
